test/e2e/network_policy.go

```
@@ -0,0 +1,130 @@
+	"context"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	imageRegistryNamespace         = "openshift-image-registry"
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofumpt)